front/app/containers/Admin/projects/project/analysis/InputPreview/components/SelectOptionText.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import useUserCustomFieldsOptions from 'api/user_custom_fields_options/useUserCustomFieldsOptions';
+
+import T from 'components/T';
+
+export const SelectOptionText = ({
+  customFieldId,
```

Comment:
No changes made to this component - just moved it into it's own file instead of being in the LongFieldValue file.